tests/shutdown/cleanup_before_shutdown.pm

```diff
@@ -21,6 +21,13 @@ use version_utils;
 
 sub run {
     select_console('root-console');
+    # Collect detailed logs to investigate shutdown issues and redirect them to serial console.
+    # Please see https://freedesktop.org/wiki/Software/systemd/Debugging/#index2h1 for the details.
+    # Boot options that are required to make logs more detalized are located in 'bootloader_setup.pm'
+    if (get_var('DEBUG_SHUTDOWN')) {
+        assert_script_run "echo -e '#!/bin/sh\\ndmesg > /dev/$serialdev' > /usr/lib/systemd/system-shutdown/debug.sh";
+        assert_script_run("chmod +x /usr/lib/systemd/system-shutdown/debug.sh");
```

Comment:
Sorry for being picky, but this looks like this to me:
![image](https://user-images.githubusercontent.com/28305643/43899751-33da92be-9be3-11e8-8985-e1c6bbe3001c.png)
Could you use same syntax in both calls with () or without them? =D

Reply:
Agree:) Changed.